Math Magicians: setup

SlowKingV opened the pull request:
## Description
This is the first project of the Math Magicians application. You will set up the environment and tools needed to develop a React application. In the following projects, you will develop the actual application.

## Project Requirements
- First, make sure that you have everything installed on your machine.
- The easiest way to create a React application is using **Create React App** (CRA) and following the instructions in this [guide](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
- The application should run in the browser without errors.

SlowKingV committed:
Install linters (ESLint & Stylelint)

SlowKingV committed:
fix linter problems

SlowKingV committed:
Fix css lint errors

SlowKingV committed:
Update README file

topeogunleye reviewed (commented):
Hi @SlowKingV, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses the old readme template. Please kindly edit your readme template to the new Microverse [README](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

SlowKingV committed:
Update README.md file to lastes template

SlowKingV committed:
Add 'WIP' status to live demo in README

The project hasn't been deployed yet, so there's no live demo to show at the moment.

SlowKingV commented:
The README has been updated to match the new template now.
I didn't realize there was a new one since I just came back from a break. Thanks for pointing it out! 🙌

anagudelogu commented on README.md:
```diff
@@ -29,42 +141,59 @@ Your app is ready to be deployed!
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](./LICENSE) licensed.
```
- [ ] Great job working on your project documentation! 🚀 However, your LICENSE link is not working because you don't have one at the moment. Please follow [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) in order to add a license to your repo! Microverse suggests that you use the MIT one, but feel free to choose. 🤓 🔧

anagudelogu reviewed (changes requested):
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @SlowKingV , 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 

<img src="https://media2.giphy.com/media/3ohhwi25ISXC7Z4tMs/giphy-downsized-medium.gif"/>

## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

topeogunleye commented:
> The README has been updated to match the new template now. I didn't realize there was a new one since I just came back from a break. Thanks for pointing it out! 🙌

@SlowKingV you're welcome

SlowKingV committed:
Add LICENSE to the repo

SlowKingV committed:
Merge branch 'setup' of github.com:SlowKingV/math-magicians2 into setup

SlowKingV commented:
I have added the LICENSE. The link seems to be working now.

cynthiainga reviewed (approved):
## STATUS APPROVED ✔️ ✔️ ✔️ 

Hi @SlowKingV, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
